content/about.md

```diff
@@ -1,49 +1,32 @@
----
+te---
 Title: "About"
 ---
-{{< project >}}
+OpenTelemetry provides the libraries, agents, and other components that you need to capture critical telemetry your production services so that you can better observe, manage, and debug them. Specifically, OpenTelemetry captures metrics, distributed traces, and resource metadata from your backend and client applications and then sends this data to backends like Prometheus, Jaeger, Zipkin, [and others](https://opentelemetry.io/registry/?s=exporter) for processing. OpenTelemetry is composed of the following:
+
+- One API and SDK per language, which include the interfaces and implementations that you need to define and create distributed traces and metrics, manage sampling and context propagation, etc.
+- Language-specific integrations for popular web frameworks, storage clients, RPC libraries, etc. that (when enabled) automatically capture relevant traces and metrics and handle context propagation
```

Comment:
should auto-instrumentation be a separate item or it's included in integrations?

Reply:
I'll add it as a separate line

Reply:
```suggestion
- Language-specific integrations for popular web frameworks, storage clients, RPC libraries, etc. that (when enabled) automatically capture relevant traces and metrics and handle context propagation
- Automatic instrumentation agents that can collect telemetry from some applications without requiring code changes
```